src/analyze.ts

```diff
@@ -208,14 +208,21 @@ export async function runQueries(
         statusReport[`analyze_custom_queries_${language}_duration_ms`] =
           new Date().getTime() - startTimeCustom;
       }
+      logger.endGroup();
+      logger.startGroup(`Interpreting results for ${language}`);
       const startTimeInterpretResults = new Date().getTime();
       const sarifFile = path.join(sarifFolder, `${language}.sarif`);
-      await runInterpretResults(language, querySuitePaths, sarifFile);
+      const analysisSummary = await runInterpretResults(
+        language,
+        querySuitePaths,
+        sarifFile
+      );
       await injectLinesOfCode(sarifFile, language, locPromise);
       statusReport[`interpret_results_${language}_duration_ms`] =
         new Date().getTime() - startTimeInterpretResults;
-      printLinesOfCodeSummary(logger, language, await locPromise);
       logger.endGroup();
+      printLinesOfCodeSummary(logger, language, await locPromise);
+      logger.info(analysisSummary);
```

Comment:
Can you flip the output here?

```suggestion
      logger.info(analysisSummary);
      printLinesOfCodeSummary(logger, language, await locPromise);
```